ingestion/src/metadata/ingestion/source/athena.py

```diff
@@ -45,6 +45,8 @@ def get_connection_url(self):
 
         return url
 
+    def fetch_sample_data(self, schema: str, table: str, connection):
```

Comment:
@ayush-shah let's not create abstract methods in each connector. All we need is a database.tablename for most of the connectors we can use "{database}.{tablename}" it looks like the only exception is bigquery.  Let's remove these from connectors and add a 
get_table_fqdn method which will return "{database}.{tablename}" whereas for big query we can override. We can use this return fqdn in sample_data to query